### PR TITLE
Added the interaction with config ms

### PR DESCRIPTION
**Pull request by ayancovich**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 spring.application.name=gateway
```
Comment (en-nbantar) [inline src/main/resources/application.properties]: What's with the "^M" on this file? I'm not sure what it means and if it should be there.

File: src/main/resources/bootstrap.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#cloud config for gateway
+spring:  cloud:    config:      uri:      - http://localhost:8086/
```
Comment (en-nbantar) [inline src/main/resources/bootstrap.yml]: Same as above.

File: Dockerfile [hunk 3]
```diff
@@ -7,7 +7,9 @@ RUN mvn package
 
 FROM openjdk:12 as Target
 COPY --from=builder /build/target/gateway-1.0.0.jar gateway.jar 
-ENV server.url=pad-b-registry
+ENV server.url=pad-b-registry /
```
Comment (nestorreloaded) [inline Dockerfile:10]: Maybe this is an encoding issue

File: src/main/resources/application.properties (hunk 1 above)
Comment (nestorreloaded) [reply src/main/resources/application.properties]: I think they are enconding issues

**Review by nestorreloaded — CHANGES_REQUESTED** [review]
Please fix the encoding issues

File: src/main/resources/bootstrap.yml (hunk 2 above)
Comment (ayancovich) [reply src/main/resources/bootstrap.yml]: Fixed. Was the line delimiters configuration problem.

File: src/main/resources/application.properties (hunk 1 above)
Comment (ayancovich) [reply src/main/resources/application.properties]: Fixed. Was the line delimiters configuration problem.

**Review by nestorreloaded — APPROVED** [review]

**Review by en-nbantar — APPROVED** [review]

**Pull request merged**